docs/source/release_notes/release-notes.rst

```diff
@@ -89,6 +89,11 @@ develop
            should improve targeted sweep throughput without adding additional load on Timelock for places where the downtime between targeted sweep iterations is a bottleneck.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/4086>`__)
 
+    *    - |devbreak|
+         - ``KeyValueService`` implementations now have a new endpoint ``deleteRows()`` that allows row-level deletes to be performed.
+           Users who extend ``KeyValueService`` will need to implement this method.
```

Comment:
but can refer to AbstractKVS for a functional implementation?

Reply:
yeah, will add this.